utils/libgds.py

```diff
@@ -0,0 +1,122 @@
+# Standard imports
+import logging
+import requests
+from tempfile import NamedTemporaryFile
+from datetime import datetime
+
+# ICA imports
```

Comment:
Ah. Just suggestion, I would name this util module as `gds.py` (which denote app aspect of it) and, to avoid confusion with SDK library naming.